pom.xml

```diff
@@ -34,7 +34,7 @@
   <properties>
     <stack.version>3.4.0-SNAPSHOT</stack.version>
     <asciidoc.dir>${project.basedir}/src/main/asciidoc</asciidoc.dir>
-    <infinispan.version>8.2.5.Final</infinispan.version>
+    <infinispan.version>9.0.0.Beta1</infinispan.version>
```

Comment:
Sure there were some issues with resolving Beta2 in maven central so I was just playing it safe [1].  I will put in Beta2 though.

[1] http://lists.jboss.org/pipermail/infinispan-dev/2017-January/017149.html